src/ws/test-remotectlcertificate.c

```diff
@@ -99,6 +102,9 @@ setup (TestCase *tc,
       g_assert (g_chmod (tc->cert_dir, 0555) == 0);
     }
 
+  if (fix->needs_openssl && !have_openssl)
+    return;
```

Comment:
Surprised to discover that it's not possible to `skip()` inside of a setup function.

Reply:
Yeah, I found that quite annoying as well..